week-1/H-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
```

Comment:
`total` what? :) Think about naming your variables meaningfully

Reply:
@martynamaron  I have renamed it groundTotal

Reply:
You mean `grandTotal`? The meaning of it is: *the final amount after everything is added up; the sum of other totals.*

I'd name it just `everyone` or `studentsAndMentors`. Doesn't need to be fancy, just descriptive! :)

Reply:
oops, yeah I mean grandTotal, will make the necessary correction, thanks